doc/developer-notes.md

```diff
@@ -1156,7 +1156,7 @@ namespace {
   `#include "primitives/transactions.h"` quote syntax.
 
   - *Rationale*: Bracket syntax is less ambiguous because the preprocessor
-    searches a fixed list of include directories without taking location of the
+    searches a fixed list of included directories without taking location of the
```

Comment:
The change from 'include directories' to 'included directories' is incorrect and introduces a grammatical error. The correct term is 'include directories'.